launch/waypoints_test.launch

```diff
@@ -8,7 +8,9 @@
     </include>
 
     <!-- This parameter must be true, overrides whatever's in morse.yaml -->
-    <param name="sim/ground_truth_localization" value="true" />
+    <!--<param name="sim/ground_truth_localization" value="true" />-->
```

Comment:
I didn't mean to include this change, but I think it's best to just remove the parameter from this launch file and use robot_localization.